CHANGELOG.rst

```diff
@@ -14,10 +14,19 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[2.0.0] - 2020-07-21
+~~~~~~~~~~~~~~~~~~~~
+
 Removed
 _______
 
-* Support for Python 3.5.  Versions 3.5.3 and above will likely still work for now, but they are no longer being tested; this lets us upgrade some dependencies and avoid confusion when aiohttp fails to install under 3.5.2 and below.  Python 3.5 reaches EOL in 1.5 months anyway.
+* Support for Python 3.5.  Versions 3.5.3 and above will likely still work for now, but they are no longer being tested; this lets us upgrade some dependencies and avoid confusion when aiohttp fails to install under 3.5.2 and below.  Python 3.5 reaches EOL in 1 month anyway.
```

Comment:
Do you mind adding `Removed` here as well? I know it is repetitive, but I had missed the title and read this as adding support.
```
* Removed support for Python 3.5.  ...
```